### PR TITLE
encrypt blockdevice mappings in worker nodes

### PR DESCRIPTION
**Pull request by xh3b4sd**
Fixes https://github.com/giantswarm/giantswarm/issues/7809.

### CODE REVIEW

File: pkg/project/project.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "8.0.3-dev"
+	bundleVersion        = "8.0.3-xh3b4sd"
```
Comment (calvix) [inline pkg/project/project.go]: i thnik this should be changed

**Comment by xh3b4sd** [comment]
Tested this on `gauss` and `giraffe`. See also https://gigantic.slack.com/archives/C7865GLSY/p1579522554002000. Upgrades are fine because the Node Pool volumes are ephemeral.

File: pkg/project/project.go (hunk 1 above)
Comment (xh3b4sd) [reply pkg/project/project.go]: Reverted.

Comment (xh3b4sd) [reply pkg/project/project.go]: Reverted.

**Comment by paurosello** [comment]
This will end up in release 10.x right?

**Comment by xh3b4sd** [comment]
Yes, it got merged to master and will be part of the next release.

**Pull request merged**